x-pack/test/case_api_integration/common/lib/authentication/index.ts

```diff
@@ -24,11 +24,23 @@ export const createSpaces = async (getService: CommonFtrProviderContext['getServ
   }
 };
 
-const createUsersAndRoles = async (getService: CommonFtrProviderContext['getService']) => {
+export const createUsersAndRoles = async (
```

Comment:
@cnasikas let me know what you think of my changes here. Basically the approach I took was when the spaces plugin is disabled we can pass in `['*']` so each role has access to all spaces (in reality that'll only be the `default` space because the spaces plugin is disabled). If we don't give each role access to the default space, then the tests won't work because the space they do have access to (e.g. `space1` etc) don't exist.